### PR TITLE
PublicSuffixList: Fix naming of "public suffix plus one".

### PR DESCRIPTION
**Pull request by pocmo**
Mixed up "eTLD" and "registrable domain".

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: components/lib/publicsuffixlist/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixList.kt [hunk 1]
```diff
@@ -59,7 +59,7 @@ class PublicSuffixList(
     }
 
     /**
-     * Returns the public suffix and one more level; known as the effective top-level domain (eTLD). Returns `null` if
+     * Returns the public suffix and one more level; known as the registrable domain domain. Returns `null` if
```
Comment (Amejia481) [inline components/lib/publicsuffixlist/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixList.kt]: I guess this is mistake domain domain?

Comment (pocmo) [reply components/lib/publicsuffixlist/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixList.kt]: Oh yes, I fixed that locally but apparently I didn't push :D

Comment (Amejia481) [reply components/lib/publicsuffixlist/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixList.kt]: 👍

Comment (pocmo) [reply components/lib/publicsuffixlist/src/main/java/mozilla/components/lib/publicsuffixlist/PublicSuffixList.kt]: Pushed :)

File: components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt [hunk 2]
```diff
@@ -87,7 +87,7 @@ interface Toolbar {
     /**
      * Adds an action to be displayed on the right side of the URL in display mode.
      *
-     * Related:
+     * Related:Pu
```
Comment (Amejia481) [inline components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt]: Nit: `Pu`?

Comment (pocmo) [reply components/concept/toolbar/src/main/java/mozilla/components/concept/toolbar/Toolbar.kt]: What the ...

**Pull request merged**